src/components/edit-movie/edit-movie.tsx

```diff
@@ -0,0 +1,106 @@
+import classNames from 'classnames';
+import { AppButton } from 'components/app-button/app-button';
+import { Genres } from 'components/menu/menu';
+import { Modal } from 'components/modal/modal';
+import styles from './edit-movie.module.scss';
+
+const genres = [Genres.COMEDY, Genres.CRIME, Genres.DOCUMENTARY, Genres.HORROR];
+
+interface Props {
+  title: string;
+  onSubmit: () => void;
+}
+
+export const EditMovie = ({ title, onSubmit }: Props) => {
+  const handleSubmit = () => {
+    onSubmit();
+    console.log('submited');
+  };
+  const handleReset = () => {
+    onSubmit();
```

Comment:
Hm-hm, is `submit` really suitable to `reset`? In my opinion submit is an action that should confirm and save your updated data and reset should just get back to previous value. If you want to pass custom callback for reset it would be better to name it `reset` as well

Reply:
Yes, it's confusing, but onSubmit for now just closes a window, I think I should not create 2 similar functions, I will rename a function instead for now

Reply:
Yes, this is better, thank you